app/models/spree_amazon/configuration.rb

```diff
@@ -0,0 +1,10 @@
+class SpreeAmazon::Configuration < Spree::Preferences::Configuration
+  preference :client_id, :string
+  preference :merchant_id, :string
+  preference :aws_access_key_id, :string
+  preference :aws_secret_access_key, :string
+
+  def use_static_preferences!
+    raise "SpreeAmazon::Configuration cannot use static preferences"
+  end
```

Comment:
I just don't understand why this is the case with the static preferences. I mean, aren't the spree preferences the same? - they are in the DB, but as the Rails app loads - they are loaded as sort of class variables.

I guess it's not a deal at all here as they are only used on the checkout, but just curious if you happen to know the answer.